docs/Installation.rst

```diff
@@ -318,6 +318,18 @@ Example usage, with the relevant output at the start of stage 1 of the bootstrap
           Source tarballs for all three EasyBuild components *must* be provided when ``$EASYBUILD_BOOTSTRAP_SOURCEPATH``
           is defined, however.
 
+.. _updating:
+
+Updating an existing EasyBuild installation
+-------------------------------------------
+
+To upgrade to a newer EasyBuild version (say, *x.y.z*) than the one currently installed (say, *a.b.c*), there are several options:
```

Comment:
rather than using `*x.y.z*`, you can use `|version|`, which will be replaced with the latest EasyBuild version (as defined in `conf.py`)